--- a/rootfs/usr/bin/btuart
+++ b/rootfs/usr/bin/btuart
@@ -1,6 +1,7 @@
 #!/bin/sh
 
-HCIATTACH=/usr/bin/hciattach
+# HCIATTACH=/usr/bin/hciattach
+BTATTACH=/usr/local/bin/btattach
 SERIAL=`cat /proc/device-tree/serial-number | cut -c9-`
 B1=`echo $SERIAL | cut -c3-4`
 B2=`echo $SERIAL | cut -c5-6`
@@ -13,10 +14,13 @@
 if [ "$uart0" = "$serial1" ] ; then
 	uart0_pins="`wc -c /proc/device-tree/soc/gpio@7e200000/uart0_pins/brcm\,pins | cut -f 1 -d ' '`"
 	if [ "$uart0_pins" = "16" ] ; then
-		$HCIATTACH /dev/serial1 bcm43xx 3000000 flow - $BDADDR
+#		$HCIATTACH /dev/serial1 bcm43xx 3000000 flow - $BDADDR
+		$BTATTACH -B /dev/serial1 -P bcm -S 3000000 &
 	else
-		$HCIATTACH /dev/serial1 bcm43xx 921600 noflow - $BDADDR
+#		$HCIATTACH /dev/serial1 bcm43xx 921600 noflow - $BDADDR
+		$BTATTACH -B /dev/serial1 -P bcm -S 921600 -N  &
 	fi
 else
-	$HCIATTACH /dev/serial1 bcm43xx 460800 noflow - $BDADDR
+#	$HCIATTACH /dev/serial1 bcm43xx 460800 noflow - $BDADDR
+	$BTATTACH -B /dev/serial1 -P bcm -S 460800 -N  &
 fi
